hs_model_program/views.py

```diff
@@ -37,10 +37,10 @@ def get_model_metadata(request):
                 software_language=mpmeta.modelProgramLanguage,
                 operating_sys=mpmeta.modelOperatingSystem,
                 url = protocol+"://"+request.get_host()+"/resource/"+resource_id+"/",
-                modelEngine = mpmeta.modelEngine.split(';'),
-                modelSoftware=mpmeta.modelSoftware.split(';'),
-                modelDocumentation=mpmeta.modelDocumentation.split(';'),
-                modelReleaseNotes=mpmeta.modelReleaseNotes.split(';'),
+                modelEngine = mpmeta.get_engine_list(),
```

Comment:
@pkdash in the javascript, each of the multiselect elements is looped through (starting line 58). These multiselect boxes are for the different model file metadata elements (i.e. `modelEngine`, `modelDocumentation`, `modelReleaseNotes`, `modelEngine`). The corresponding files names stored under these terms come from the AJAX  data (line 64). Then the file names are added to the multiselect (starting in line 68).